src/lib/operators/join_hash.cpp

```diff
@@ -119,6 +119,8 @@ using Partition = std::conditional_t<std::is_trivially_destructible_v<T>, uninit
 using SmallPosList = boost::container::small_vector<RowID, 1>;
 
 template <typename T>
+// In case we consider runtime to be more relevant, the flat hash map
+// is a better performing but significantly larger alternative to the bytell map.
```

Comment:
Do you have numbers for that?

Reply:
It's single digit percent faster, but within Hyrise it's barely noticable from what I measured. In the long run, smaller tables should lead to larger radix hash maps which will still be L2 resident. So when we really optimize for a particular cache size the bytell should have an advantage for large joins (which are radix partitioned).

Reply:
Could you include that in the comment? Currently, the comment reads as if there might be more significant performance gains but we want to save a handful of DIMMs. ;)